pkg/vpcmodel/grouping.go

```diff
@@ -461,6 +461,15 @@ func (g *GroupConnLines) String() string {
 	return strings.Join(linesStr, "\n") + "\n"
 }
 
+// get the grouped connectivity stateLessness
+func (g *GroupConnLines) HasStatelessConns() bool {
```

Comment:
the function is called only within this pkg, so should not be exported (rename to `hasStatelessConns`).. same for the `string()` function